src/formatter-settings/index.ts

```diff
@@ -61,6 +62,14 @@ export class JavaFormatterSettingsEditorProvider implements vscode.CustomTextEdi
         vscode.commands.executeCommand("vscode.openWith", filePath, "java.formatterSettingsEditor");
     }
 
+    public reopenWithTextEditor = instrumentOperation("formatter.showTextEditor", (operationId: string, uri: any) => {
+        sendInfo(operationId, { editor: "RawTextEditor" });
+        if (uri instanceof vscode.Uri) {
+            this.webviewPanel?.dispose();
```

Comment:
What will happen if we do not dispose it?

Reply:
We'll have another raw text editor open, without closing the current custom editor.

Reply:
Update: VSCode behavior is to open another raw editor without closing the current editor, we'd better follow VSCode's.